CMakeLists.txt

```diff
@@ -125,60 +125,67 @@ endif() # TRITON_ENABLE_GPU
 #
 configure_file(src/libtriton_pytorch.ldscript libtriton_pytorch.ldscript COPYONLY)
 
-if (${TRITON_PYTORCH_DOCKER_BUILD})
-  if (CMAKE_HOST_SYSTEM_PROCESSOR MATCHES "aarch64")
-    set(LIBS_ARCH "aarch64")
-    set(CONDA_LIBS
-        "libopenblas.so.0"
-    )
-  else()
-    set(LIBS_ARCH "x86_64")
-    set(CONDA_LIBS
-        "libmkl_core.so"
-        "libmkl_gnu_thread.so"
-        "libmkl_intel_lp64.so"
-        "libmkl_intel_thread.so"
-        "libmkl_def.so"
-        "libmkl_vml_def.so"
-        "libmkl_rt.so"
-        "libmkl_avx2.so"
-        "libmkl_avx512.so"
-        "libmkl_sequential.so"
-        "libomp.so"
-    )
-  endif()
+set(PT_LIBS
+    "libc10.so"
+    "libc10_cuda.so"
+    "libtorch.so"
+    "libtorch_cpu.so"
+    "libtorch_cuda.so"
+    "libtorch_global_deps.so"
+)
+
+if (${TRITON_PYTORCH_ENABLE_TORCHVISION})
   set(PT_LIBS
-      ${CONDA_LIBS}
-      "libc10.so"
-      "libc10_cuda.so"
-      "libtorch.so"
-      "libtorch_cpu.so"
-      "libtorch_cuda.so"
-      "libtorch_global_deps.so"
+      ${PT_LIBS}
       "libtorchvision.so"
   )
-  set(OPENCV_LIBS
-      "libopencv_video.so"
-      "libopencv_videoio.so"
-      "libopencv_highgui.so"
-      "libopencv_imgcodecs.so"
-      "libopencv_imgproc.so"
-      "libopencv_core.so"
-      "libpng16.so"
+endif() # TRITON_PYTORCH_ENABLE_TORCHVISION
+
+if (${TRITON_PYTORCH_ENABLE_TORCHTRT})
+  set(PT_LIBS
+      ${PT_LIBS}
+      "libtorchtrt_runtime.so"
   )
+endif() # TRITON_PYTORCH_ENABLE_TORCHTRT
 
-  string(REPLACE ";" " " CONDA_LIBS_STR "${CONDA_LIBS}")
+if (CMAKE_HOST_SYSTEM_PROCESSOR MATCHES "aarch64")
+  set(LIBS_ARCH "aarch64")
+  set(CONDA_LIBS
+      "libopenblas.so.0"
+  )
+else()
+  set(LIBS_ARCH "x86_64")
+  set(CONDA_LIBS
```

Comment:
CONDA_LIBS -> PT_CPU_LIBS

Reply:
Technically this is for all dependencies that are inside the conda package/directory. Not just CPU libs. I would like to keep the name generic for this reason.